L/libunicorn/build_tarballs.jl

```diff
@@ -0,0 +1,36 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libunicorn"
+version = v"1.1.0"
```

Comment:
Where does this version number come from?

Reply:
Also wondering about this... on https://github.com/unicorn-engine/unicorn they released 1.0.2 just 11 days ago. If for some reason there is need to make an "interim release", I'd suggest to still call it 1.0.2, or perhaps 1.1.0-dev (i.e. a prerelease version, in case there's a real 1.1.0 later on you also want to wrap).

Reply:
I'll go with 1.0.1-dev, I suppose, and will bump it up to 1.0.2 once Unicorn merges a patch I just submitted, which fixes a race condition bug.

Reply:
Please, no, don't use pre-release bit.  Better to use `1.0.2`

Reply:
sure thing